src/sparse_ci/ci_reference.cc

```diff
@@ -518,6 +504,24 @@ void CI_Reference::build_cas_reference(std::vector<Determinant>& ref_space) {
     outfile->Printf("\n  Reference generated from %d MOs", na);
 }
 
+void CI_Reference::build_cas_reference_full(std::vector<Determinant>& ref_space) {
```

Comment:
```suggestion
void CI_Reference::build_cas_reference(std::vector<Determinant>& ref_space) {
```
I am not sure we need the work `full`, unless it distinguishes this function from another one.

Reply:
Yes, there is a distinction between the two. The original one only selects up to the `subspace_size_`. The full one gives all the CAS determinants.